lib/hydra/version.rb

```diff
@@ -1,3 +1,3 @@
 module Hydra
-  VERSION = "10.0.1".freeze
+  VERSION = "11.0.0".freeze
```

Comment:
Happy to make it rc1. Not sure what the QA process on this is. I've run through the tutorial with it but I'd be happy to see more testing done.